godot-core/src/registry.rs

```diff
@@ -327,6 +450,18 @@ fn register_class_raw(info: ClassRegistrationInfo) {
     assert!(!info.is_editor_plugin);
 }
 
+fn unregister_class_raw(class_name: &ClassName) {
+    out!("Unregister class: {class_name}");
+    unsafe {
+        #[allow(clippy::let_unit_value)]
+        let _: () = interface_fn!(classdb_unregister_extension_class)(
+            sys::get_library(),
+            class_name.string_sys(),
+        );
+    }
+    godot_print!("Class {class_name} unloaded");
```

Comment:
Sorry if a dumb question (and not sure it's really worth opening a new issue for so I thought I'd do a small comment here)

Is there a way to suppress these lines from printing? 

I have an application that I use for integration tests for my project (launched via godot cli) and with this new change I now get a big list of nodes at the end of the output which isn't ideal for me.